cedar-policy-cli/src/lib.rs

```diff
@@ -316,10 +316,10 @@ pub struct AuthorizeArgs {
 
 #[derive(Clone, Copy, Debug, Default, Eq, PartialEq, ValueEnum)]
 pub enum PolicyFormat {
-    /// The standard human-readable Cedar policy format, documented at https://docs.cedarpolicy.com/policies/syntax-policy.html
+    /// The standard human-readable Cedar policy format, documented at <https://docs.cedarpolicy.com/policies/syntax-policy.html>
     #[default]
     Human,
-    /// Cedar's JSON policy format, documented at https://docs.cedarpolicy.com/policies/json-format.html
+    /// Cedar's JSON policy format, documented at <https://docs.cedarpolicy.com/policies/json-format.html>
```

Comment:
These strings appear verbatim in the CLI `--help`. I don't see any harm in including the angle brackets there, but thought it was worth pointing out.

Reply:
If we don't add the angle-brackets it won't be a clickable link in the docs

Reply:
This is in CLI, so I don't expect too many people to be looking at them

Reply:
Fair enough, I can role that back